elastic_apm.services.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  logger.channel.elastic_apm:
+    class: Drupal\Core\Logger\LoggerChannel
+    factory: logger.factory:get
+    arguments: ['elastic_apm']
+
+  http_middleware.elastic_apm:
+    class: Drupal\elastic_apm\StackMiddleware\ElasticApmMiddleware
+    tags:
+      - { name: http_middleware, priority: 350 }
+
+  elastic_apm.service:
+    class: Drupal\elastic_apm\ElasticApm
+    arguments:
+      - '@config.factory'
+      - '@current_user'
+
+  elastic_apm.request_event_subscriber:
+    class: Drupal\elastic_apm\EventSubscriber\ElasticApmRequestSubscriber
```

Comment:
**CHANGE REQUEST**

No need to prefix the class names with `ElasticApm`, they're already namespaced. Subscriber and middleware.